docs/rules/0122/resource-collection-identifiers.md

```diff
@@ -0,0 +1,71 @@
+---
+rule:
+  aip: 122
+  name: [core, '0122', resource-collection-identifiers]
+  summary: Resource patterns must use lowerCamelCase for collection identifiers.
+permalink: /122/resource-collection-identifiers
+redirect_from:
+  - /0122/resource-collection-identifiers
+---
+
+# Resource pattern collection identifiers
+
+This rule enforces that messages that have a `google.api.resource` annotation
+have properly formatted collection identifiers in each `pattern`, as described
+in [AIP-122][].
+
+## Details
+
+This rule scans messages with a `google.api.resource` annotation, and validates
```

Comment:
Maybe say something about the lowerCamel requirement?

Reply:
Sure thing, done.